src/importexport/musicxml/internal/musicxml/import/importmusicxmlpass2.cpp

```diff
@@ -2179,13 +2179,18 @@ void MusicXmlParserPass2::part()
     setPartInstruments(m_logger, &m_e, part, id, m_score, m_pass1.getInstrList(id), m_pass1.getIntervals(id), instruments, partName);
     partName = replacePartNameAccidentals(partName);
     part->setPartName(partName);
-    if (mxmlPart.getPrintName() && !isLikelyIncorrectPartName(partName)) {
+    const bool inconsistentVisibility = mxmlPart.getPrintName() != mxmlPart.getPrintAbbr();
+    if (!isLikelyIncorrectPartName(partName)) {
         part->setLongNameAll(partName);
     } else {
         m_pass1.getPart(id)->setLongNameAll(u"");
```

Comment:
why `m_pass1.getPart(id)` rather than `part`?

Reply:
you should ask the original implementors 😉